models/jira__daily_issue_field_history.sql

```diff
@@ -15,7 +15,7 @@
 
 {% if is_incremental() %}
 -- set max date_day with lookback as a variable for multiple uses
-{% set max_date_day = jira.jira_lookback(from_date='max(date_day)', datepart='day', interval=var('lookback_window', 3)) %}
+{% set max_date_week = jira.jira_lookback(from_date='max(date_week)', datepart='week', interval=var('lookback_window', 1)) %}
```

Comment:
Do we need to update the lookback window README section (and thus the CHANGELOG) that the default is now 1 week rather than 3 days? Or is it just for specific cases?

Reply:
Very good catch--definitely yes.  I have updated!